himan-scripts/CB-TCU-cloud.lua

```diff
@@ -46,15 +54,17 @@ end
 
 RR = luatool:Fetch(current_time, HG, param("RRR-KGM2"), current_forecast_type)
 
-CBlimit = 12  --required vertical thickness [degrees C] to consider a CB (tweak this..!)
-TCUlimit = 9  --required vertical thickness [degrees C] to consider a TCU (tweak this..!)
-CBtopLim = 263.15  --required top T [K] to consider a CB (tweakable!)
+if not NL or not NM or not RR then
+  logger:Error("Some data not found")
+  return
+end
+
+CBlimit = 2000  --required vertical thickness [degrees C] to consider a CB (tweak this..!)
```

Comment:
The comment for these variables is a leftover from earlier commit?

Also in smarttool macro TCulimit is 1500 (not 1000).